sdk-ext-ui/src/main/java/cash/z/ecc/sdk/ext/ui/regex/ZecRegex.kt

```diff
@@ -0,0 +1,32 @@
+package cash.z.ecc.sdk.ext.ui.regex
+
+import android.content.Context
+import cash.z.ecc.sdk.ext.ui.R
+import cash.z.ecc.sdk.model.MonetarySeparators
+
+object ZecRegex {
+
+    /**
+     * Builds regex with current local monetary separators for continuous input checking.
+     *
+     * Regex example: ^^$|[0-9]+([,]$|[,][0-9]+)?([.]$|[.][0-9]+)?$
+     * Inputs may differ according to user locale.
+     *
+     * Valid amounts: "" 123 123, 123. 123,456 123.456 123,456.789 etc.
+     * Invalid amounts: 123,, 123,. 123.. .123 ,123 123,456,789 123.456.789 etc.
+     *
+     * @param context used for loading localized pattern from strings.xml
+     * @param separators which consist of localized monetary separators
+     *
+     * @return regex with pattern for entered ZEC amount
+     */
+    fun getZecAmountContinuousFilter(context: Context, separators: MonetarySeparators): Regex {
```

Comment:
I like the name "continuous filter" as that does a great job describing what this method does.

Thinking about this as a "public API" (since it is in sdk-ext-ui and we hope to make it reusable), we could probably simplify the return type.  I don't think clients need to know this is implemented as a regex, and a simple boolean return type is probably good.  Following Kotlin conventions for collections.filter, a true return value means keep the item (e.g. it is good)

So what do you think of a signature like this?

`fun ZecString.filterContinuous(context: Context, separators: MonetarySeparators, zecString: String)`

Reply:
Yes, this can work. To be clear here, you propose to keep it in the `ZecRegex.kt` and make an extension function of 
`ZecString.kt` from it?